src/sage/geometry/polyhedron/base6.py

```diff
@@ -1596,10 +1596,10 @@ def affine_hull_manifold(self, name=None, latex_name=None, start_index=0, ambien
 
             sage: D = polytopes.dodecahedron()                                  # optional - sage.rings.number_field
             sage: E3 = EuclideanSpace(3)                                        # optional - sage.rings.number_field    # optional - sage.symbolic
-            sage: submanifolds = [                                              # optional - sage.rings.number_field    # optional - sage.symbolic
+            sage: submanifolds = [                                              # optional - sage.rings.number_field    # optional - sage.symbolic  # long time
```

Comment:
I'd prefer the `# long time` to appear before the `# optional - sage`. These annotations are aligned so that most of the annotation is not visible in the HTML format (one needs to scroll). But the `# long time` is for the user to see without scrolling

Reply:
Will fix